sros2/test/conftest.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Apex.AI, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+# Disable lxml2 lookup of resources on the internet by configuring it to use a proxy
+# that does not exist
+os.environ['HTTP_PROXY'] = 'http://example.com'
```

Comment:
I don't know enough about conftest.py, but with pytest unset this after the testing this package? Might be an issue if the original proxy setting, be it empty/unset or somethings else for the build farm, is not restored.

Reply:
@ruffsl `os.environ` will only modify the environment for the current process, so this will only affect the environment for pytests in the sros2 package

Reply:
I guess one other important piece of information is that `pytest` is invoked once per package, so all the other packages get their own `pytest` with a clean set of environment variables.

Reply:
Ah, TDIL. Thanks!